index.html

```diff
@@ -115,34 +115,36 @@ <h2>A project for first-time contributions</h2>
         <!-- make sure there is one line of space above and below your card -->
         <!-- ========== ⬇⬇ Paste YOUR CARD directly BELOW this line ⬇⬇ ========== -->
 
-        <div class="card">
-          <p class="name">Harshita Kohli</p>
```

Comment:
It looks like you've removed Harshita's card. Do you mind adding your card above hers instead?

Reply:
I see the issue: git merged her card into your card for some reason. This is a bit of a unique situation. Would you like assistance in recovering the card? It might also be easier if you create a new pull request on a completely new branch of this repo. It's up to you! 😃

Reply:
I have created a new pull request as suggested. Thank you.